README.md

```diff
@@ -50,6 +50,44 @@ Installing this plugin adds support for the Optimism ecosystem:
 ape console --network optimism:sepolia
 ```
 
+### OP Stack
+
+Use the `optimism` base-class in any custom networks using the OP stack.
+For example, to configure a custom network for Fraxtal network, add the following to your `pyproject.toml`
+
+```toml
+[[tool.ape.networks.custom]]
+name = "mainnet"
+ecosystem = "fraxtal"
+# Tell Ape to use optimism as the base plugin, instead of ethereum.
+base_ecosystem_plugin = "optimism"
+chain_id = 252
+
+# (optional): Configure an RPC. Else, Ape will select a random one automatically.
+[tool.ape.node.fraxtal.mainnet]
+uri = "https://rpc.frax.com"
+```
+
+Or equivalent `ape-config.yaml`:
+
+```yaml
+networks:
+  custom:
+    - name: mainnet
+      ecosystem: fraxtal
+      base_ecosystem_plugin: optimism
+
+node:
+ fraxtal:
+    mainnet:
+      uri: https://rpc.frax.com
+```
+
+There are two main benefits of using Optimism as the base-class instead of Ethereum for networks using the OP stack:
```

Comment:
@fubuloubu What other benefits can we include in the plugin?

Reply:
In terms of docs what we currently have, think we have that covered nicely

In terms of ideas, some ideas:
- track L1 block confirmations
- do native bridge actions (there are other bridge types):
  - L1 -> L2 deposits of supported tokens e.g. ETH
  - L2 -> L1 withdrawals (which take 7 days)
- they also have some native "pre-compiles" that come out of the box w/ OP stack chains